draft-ietf-quic-recovery.md

```diff
@@ -1512,24 +1519,30 @@ newly acked_packets from sent_packets.
    InCongestionRecovery(sent_time):
      return sent_time <= congestion_recovery_start_time
 
-   OnPacketsAcked(acked_packets):
-     for packet in acked_packets:
-       // Remove from bytes_in_flight.
-       bytes_in_flight -= packet.size
-       if (InCongestionRecovery(packet.time_sent)):
-         // Do not increase congestion window in recovery period.
-         return
-       if (IsAppOrFlowControlLimited()):
-         // Do not increase congestion_window if application
-         // limited or flow control limited.
-         return
-       if (congestion_window < ssthresh):
-         // Slow start.
-         congestion_window += packet.size
-         return
-       // Congestion avoidance.
-       congestion_window += max_datagram_size * acked_packet.size
-           / congestion_window
+  OnPacketsAcked(acked_packets):
+    for acked_packet in acked_packets:
+      OnPacketAcked(acked_packet)
+
+  OnPacketAcked(acked_packet):
+    // Remove from bytes_in_flight.
+    bytes_in_flight -= acked_packet.sent_bytes
+
+    // Do not increase congestion_window if application
+    // limited or flow control limited.
+    if (IsAppOrFlowControlLimited())
+        return
```

Comment:
Please change back to 2-space indent

Reply:
> Please change back to 2-space indent

@ianswett This has been fixed by @junhochoi. Thanks!